microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -127,10 +129,12 @@ The Evergreen distribution mode ensures that your WebView2 app is taking advanta
 
 When you use the Evergreen distribution mode of the WebView2 Runtime, your WebView2 app assumes that clients have the latest Runtime.  Your app cannot require a particular version of the WebView2 Runtime for all apps on the client.  By the time a new WebView2 SDK package is released, a compatible version of the WebView2 Runtime has already been distributed to clients.  Therefore it's OK for your WebView2 app to use the APIs that are in the latest version of the WebView2 SDK.
 
-See [Understanding browser versions and WebView2](versioning.md).
+See also:
+* [Understanding browser versions and WebView2](versioning.md).
+* [Delivering Microsoft Edge WebView2 Runtime to managed Windows 10 devices](https://blogs.windows.com/msedgedev/2022/12/14/delivering-microsoft-edge-webview2-runtime-to-managed-windows-10-devices/).
```

Comment:
@liminzhu - What to say here?

Rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution?branch=pr-en-us-2385#details-about-the-evergreen-runtime-distribution-mode

GitHub rendered (smaller heading sizes than LMC):
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/managed/microsoft-edge/webview2/concepts/distribution.md#details-about-the-evergreen-runtime-distribution-mode

Reply:
working session done